### PR TITLE
update branch references to `main`

### PR DESCRIPTION
**Pull request by maxcountryman**
This updates references to our mainline branch to point to `main`.

Related to: #213.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 Given that the parser produces a typed AST, any changes to the AST will technically be breaking and thus will result in a `0.(N+1)` version. We document changes that break via addition as "Added".
 
 ## [Unreleased]
-Check https://github.com/andygrove/sqlparser-rs/commits/master for undocumented changes.
```
Comment (andygrove) [inline CHANGELOG.md:9]: `andygrove` should be replaced with `ballista-compute` now as well

**Comment by andygrove** [comment]
Github actions will also need to be updated

File: CHANGELOG.md (hunk 1 above)
Comment (maxcountryman) [reply CHANGELOG.md:9]: Good catch! Updated a few references.

**Comment by Dandandan** [comment]
Changes look good to me. Ok to merge this also for you @maxcountryman ?

**Comment by maxcountryman** [comment]
> Changes look good to me. Ok to merge this also for you @maxcountryman ?

Yep, if everything looks good to everyone I think it’s ready to merge.

**Pull request merged**